### ISSUE TITLE
Does Sidekiq provide some JSON API for API only Rails apps?

### ISSUE THREAD

**Issue by exocode** [mperham/sidekiq#5317]
I have a Rails API app (6.2.x). So there are no GUIs or views.

Does Sidekiq provide some predefined implementation to respond with statistics, stats or informations for such API apps? 

And if not, is it planned or can someone provide me a entrypoint to do build such basic actions by myself?

I simply wanna know such informations which are regularly in the Sidekiq UI:

- existing queues with
- number of failed jobs 
- number of waiting jobs
- logs etc... 

Thank you very much in advance

**Comment by exocode** [comment]
I found something. I think I can create ControllerActions from that:
https://github.com/mperham/sidekiq/wiki/API

Are there predefined responses / actions like the ones who put in routes.rb? and the ActionViews (but only for API responses)? 
If not, you can quietly close that question.. Thank you

**Comment by mperham** [comment]
Correct, the Sidekiq API is what you want to be looking at. You create the JSON payloads / actions.
